.github/workflows/flow-pull-request-checks.yaml

```diff
@@ -135,7 +135,6 @@ jobs:
       - env-vars
       - unit-tests
       - e2e-tests
-      - e2e-mirror-node-tests
```

Comment:
wrong one, I meant to remove this from the needs of the e2e-relay-tests step

Reply:
Fixed